odoo/tools/translate.py

```diff
@@ -292,6 +292,49 @@ def parse_xml(text):
 def serialize_xml(node):
     return etree.tostring(node, method='xml', encoding='unicode')
 
+
+MODIFIER_ATTRS = ["invisible", "readonly", "required", "column_invisible", "attrs"]
+def transformer(term_en):
```

Comment:
I suggest you bind it to `xml_translate` and `html_translate` like 
https://github.com/odoo/odoo/blob/73bf5fbfc4b9e07667727cafcb9a314187c3e3c0/odoo/tools/translate.py#L413

Reply:
yes, I tink the transformer is only useful for xml fields

Reply:
Please find another that is not so generic.  Every pure function that takes some string and returns some string could be named "transform".
What about `xml_term_adapter` instead?  To me "adapt" suggests to make a value conform to another value.  In other words, it suggests a minimalistic transformation, which it is :wink:

Reply:
> Please find another that is not so generic. Every pure function that takes some string and returns some string could be named "transform".

Sure! This was a first draft to discuss with you. I think `xml_term_adapter` is perfect.